test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -614,17 +611,26 @@ def test_cgroup_periodic_update(self):
         self.server.status(JOB, [ATTR_o, ATTR_e], jid)
         o = j.attributes[ATTR_o]
         self.mom.log_match("%s;update_job_usage: " % jid +
+                           "CPU usage: 0.000 secs",
+                           max_attempts=2,
+                           starttime=self.server.ctime)
+        self.mom.log_match("%s;update_job_usage: " % jid +
+                           "Memory usage: mem=0b",
+                           max_attempts=2,
+                           starttime=self.server.ctime)
+        self.mom.log_match("%s;update_job_usage: " % jid +
                            "Memory usage: vmem=0b",
                            max_attempts=2,
                            starttime=self.server.ctime)
-        self.mom.log_match(".*%s;update_job_usage: " % jid,
-                           "CPU usage: .*secs.*",
+        # Allow some time to pass for values to be updated
+        time.sleep(5)
+        self.mom.log_match("%s;update_job_usage: " % jid +
+                           "CPU usage: [0-9.]+ secs",
                            regexp=True,
                            max_attempts=5,
                            starttime=self.server.ctime)
-        time.sleep(5)
-        self.mom.log_match(".*%s;update_job_usage: " % jid +
-                           "Memory usage: vmem=[1-9][0-9]+.*",
+        self.mom.log_match("%s;update_job_usage: " % jid +
+                           "Memory usage: vmem=[1-9][0-9]+kb",
                            regexp=True,
                            max_attempts=5,
                            starttime=self.server.ctime)
```

Comment:
No need for self.server.ctime in the above log_match() calls.  You have the job ID in the message.

Reply:
Updated.